src/pages/PlanPage/subRoutes/UpgradePlanPage/UpgradeForm/UpdateBlurb/UpdateBlurb.tsx

```diff
@@ -48,7 +49,7 @@ const UpdateBlurb = ({
       {diffSeats && (
         <li className="pl-2">{`You are changing seats from ${currentPlan?.planUserCount} to [${seats}]`}</li>
       )}
-      {diffBillingType && (
+      {diffBillingType && !currentIsFree && (
```

Comment:
weird this is showing up, I thought I had committed it already

Reply:
yah ikr?